src/esp/gfx/Renderer.h

```diff
@@ -16,6 +16,12 @@ namespace gfx {
 class Renderer {
  public:
   enum class Flag {
+    /**
+     * @brief No textures for the meshes.
```

Comment:
```suggestion
     * No textures for the meshes.
```

Complex reasons why is it that way, but with the `@brief` the docs won't show up in the output.

Same in `RenderTarget::Flag`.

Reply:
OK. May I ask what we shall do then.

Reply:
Just don't use `@brief` for enum values, that's it :)